packages/labs/src/components/select/select.tsx

```diff
@@ -291,10 +310,31 @@ export class Select<T> extends React.Component<ISelectProps<T>, ISelectState<T>>
     };
 
     private handleQueryChange = (event: React.FormEvent<HTMLInputElement>) => {
-        const { inputProps = {} } = this.props;
-        this.setState({ query: event.currentTarget.value });
+        const { inputProps = {}, onQueryChange } = this.props;
+        const query = event.currentTarget.value;
```

Comment:
@brieb do the tests work with `input.simulate("change", { target: { value: "..." } })` if you change this to `event.target.value`?